Makefile

```diff
@@ -30,6 +31,7 @@ NAME:=ocne
 
 GIT_COMMIT:=$(shell git rev-parse HEAD)
 BUILD_DATE:=$(shell date +"%Y-%m-%dT%H:%M:%SZ")
+CLI_VERSION:=$(shell grep Version: ${MAKEFILE_DIR}/buildrpm/ocne.spec | cut -d ' ' -f 2)
```

Comment:
It's a good idea to make sure there aren't existing commands that do what you need but capture all possible cases.  Here:

```
$ rpmspec -q --queryformat='%{VERSION}-%{RELEASE}' buildrpm/ocne.spec
2.0.4-3.el9
```

Reply:
I suppose this won't work on mac...

Reply:
Yeah, that's a good point. Since we only support running the CLI on Linux, should I change it to `rpmspec`?

Reply:
Maybe do `rpmspec` on Linux because our builds *have* to be right, and use the `grep` way on Mac?  We don't officially support Mac so it's less horrible if the answer is slightly wrong.